vnext/Microsoft.ReactNative/NativeModuleRegistry.cpp

```diff
@@ -0,0 +1,9 @@
+#include "pch.h"
+#include "NativeModuleRegistry.h"
+#if __has_include("Bridge.NativeModuleRegistry.g.cpp")
```

Comment:
curious, why do we need to do this for every generated file?

Reply:
No.  They're a byproduct of how I was adding the new classes through VS.  I can remove those.